pet-store-react-hooks/src/home/Home.tsx

```diff
@@ -0,0 +1,73 @@
+import { useCallback, useEffect, useRef, useState } from 'react';
```

Comment:
Please rename `home` to `Home` to match the component file name. Use `git mv` to rename it so that you don't have any problems with git.

Reply:
Still had problems but i renamed it to Home2 and then back to Home